TestAssets/DesktopTestProjects/DesktopAppWithNativeDep/project.json

```diff
@@ -2,7 +2,7 @@
   "version": "1.0.0-*",
   "dependencies": {
     "PackageWithFakeNativeDep": "1.0.0-*",
-    "Microsoft.NETCore.Platforms": "1.0.1"
+    "Microsoft.NETCore.Platforms": "1.0.2-servicing-24611-07"
```

Comment:
Ah, OK.  That explains it.  We had an installer fix and that's why we are shipping a new version.

Thanks.  I'll close this PR.